testing/ncids-js-testing/cypress/integration/InnerPage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Inner page template
+
+	I want to open see the example inner page
+
+	Scenario: Opening a page template
+		Given I open Example Inner Page template
```

Comment:
since cypress has a base url setup in configs, you can utilize it by passing the path in the navigation steps, such as 
Given user navigates to "/inner-pages/inner-page.html"